### PR TITLE
Feature/wcm 218 broad assessment of missing features

### PR DESCRIPTION
**Pull request by rossmiller**
Various fixes.

### CODE REVIEW

File: file_upload_handler/test/test_s3.py [hunk 1]
```diff
@@ -58,49 +56,6 @@ def test_chunk_is_received(self, client):
         # Check that we started to send data
         self.s3_file_handler.executor.mock_calls[0] = call.send(b"4")
 
-    @patch("file_upload_handler.s3.boto3_client")
```
Comment (rossmiller) [inline file_upload_handler/test/test_s3.py:61]: Reinstate

File: import_wordpress/utils/block_content.py [hunk 2]
```diff
@@ -120,30 +120,41 @@ def process_image(img, wp_attachments):
...
+
+
 def append_block_text(blocks, parent_tags):
-    proceeding_html = flatten_parent_tags(parent_tags)
-    text_content = proceeding_html.replace("<body >", "").replace("</body>", "")
+    proceeding_html = sanitize(flatten_parent_tags(parent_tags))
+    text_content = get_body_contents(proceeding_html)
 
-    text_content_no_lb = proceeding_html.replace("\n", " ").replace("\r", "")
+    text_content = re.sub(" +", " ", text_content)
```
Comment (rossmiller) [inline import_wordpress/utils/block_content.py:147]: This is the bit!

Comment (SamDudley) [reply import_wordpress/utils/block_content.py:147]: Check my commit for a suggested change.

**Review by SamDudley — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**